src/site/content/en/blog/introducing-tooling-report/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Choose the best build tool for your project with tooling.report
+subhead: Select and configure build tools based on best practices.
+description: |
+  Select and configure build tools based on best practices.
+scheduled: true
+date: 2020-06-29
+hero: hero.jpg
+alt: A description of the hero image for screen reader users.
+tags:
+  - blog
+  - javascript
+  - css
+  - html
```

Comment:
We don't have an `html` tag under https://github.com/GoogleChrome/web.dev/blob/master/src/site/_data/postTags.js

We could potentially add a `tooling-report` tag although that might be premature

It seems like we should have a generic `tooling` tag or `build-tools` tag or something like that

I'll leave it up to someone else to update postTags.js

Reply:
cc @jpmedley

Reply:
I like your idea. I’ll add a commit to this PR for the `postTags` changes.

```suggestion
tags:
  - blog
  - javascript
  - css
  - html
  - tooling
```

Reply:
How many tooling articles do we have? If there are like 3, it's probably too soon to break them down by type. 

html is one we should probably have a tag for too. Be sure to add that.